swarmlib/gwo/gwo_problem.py

```diff
@@ -0,0 +1,60 @@
+# ------------------------------------------------------------------------------------------------------
+#  Copyright (c) Leo Hanisch and contributors. All rights reserved.
+#  Licensed under the BSD 3-Clause License. See LICENSE.txt in the project root for license information.
+# ------------------------------------------------------------------------------------------------------
+
+import logging
+from copy import deepcopy
+from .wolf import Wolf
+from .visualizer import Visualizer
+
+# pylint: disable=too-many-instance-attributes
+
+LOGGER = logging.getLogger(__name__)
+
+
+class GWOProblem:
+    def __init__(self, **kwargs):
+        """
+        Initialize a new grey wolf optimization problem.
+        """
+
+        self.__iteration_number = kwargs.get('iteration_number', 20)
```

Comment:
The API default values should be the same like when invoking the library via the CLI. In this case `20` (API) vs. `30` (CLI)